web/src/__snapshots__/AlertPane.test.tsx.snap

```diff
@@ -35,7 +35,7 @@ exports[`renders multiple lines of a crash log 1`] = `
             dateTime="1949-11-18T09:39:00.000Z"
             title="1,555,970,585,039"
           >
-            67yr
+            67yr ago
```

Comment:
(it's also not strictly necessarily to leave the "ago" in here, i put it in to differentiate "build duration" from "time elapsed since last build", happy for another way to differentiate these that doesn't take up space in the sidebar!)

Reply:
Yeah, I also find that "ago" is useful for distinguishing two tiny timestamps at a glance.